packages/realm-web/src/tests/WatchStream.test.ts

```diff
@@ -692,14 +687,8 @@ describe("WatchStream", () => {
             // This is a deviation from the spec. We do not support the legacy macOS < 10 CR-only newlines.
             // The server does not generate them, and there would be some overhead to supporting them.
             ws.feedBuffer(toBuffer('event: message\rdata: {"a": 1}\r\r'));
-            if (true) {
-                // This is what we do.
-                assert.deepEqual(ws.state, WatchStreamState.NEED_DATA);
-            } else {
-                // This is what we would do if following the spec.
```

Comment:
So we're not following the spec?

Reply:
I think the "spec" referred to here is the "server-sent events" / "event source" specification. The comment above states that the server won't generate these and since we're only implementing towards this particular server, I think it's safe enough to leave it at this.

@RedBeard0531 would know the details of this comment, but for now I just wanted to remove the `if (true)` since that triggered a lint error.